CMakeLists.txt

```diff
@@ -153,6 +153,17 @@ set(Slicer_EXTENSION_SOURCE_DIRS
 #  )
 #list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
 
+#SlicerMorph modules
```

Comment:
Note for later, comments should include a space (so that they are consistent with remaining of the code base):
```
# SlicerMorph modules
```